code-challenges.test.js

```diff
@@ -36,16 +66,25 @@ const object3 = { number: -7 }
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+//divisibleByThree function takes an object as an argument 
+//Checks against the modulo
+//It returns the appropriate string
```

Comment:
You can elaborate this pseudo code like this:
- input: object
- output: string
- access value using dot notation and assign it to a variable
- check if the number has a remainder of 0 when divided by 3 using modulo operator
- if so, return string interpolation saying the number is divisible by three
- if not, string interpolation saying number is not divisible by three